src/renderer/Renderer.js

```diff
@@ -95,52 +137,20 @@ export default class Renderer {
         gl.bindTexture(gl.TEXTURE_2D, this.zeroTex);
     }
 
-    /**
-    * Get Renderer visualization center
-    * @return {RPoint}
-    */
-    getCenter () {
-        return { x: this._center.x, y: this._center.y };
-    }
-
-    /**
-     * Set Renderer visualization center
-     * @param {number} x
-     * @param {number} y
-     */
-    setCenter (x, y) {
-        this._center.x = x;
-        this._center.y = y;
-    }
-
-    /**
-     * Get Renderer visualization bounds
-     * @return {*}
-     */
-    getBounds () {
-        const center = this.getCenter();
-        const sx = this.getZoom() * this.getAspect();
-        const sy = this.getZoom();
-        return [center.x - sx, center.y - sy, center.x + sx, center.y + sy];
-    }
-
-    /**
-     * Get Renderer visualization zoom
-     * @return {number}
-     */
-    getZoom () {
-        return this._zoom;
+    _initZoom (map) {
+        const b = map.getBounds();
+        const nw = b.getNorthWest();
+        const sw = b.getSouthWest();
+        const z = (util.projectToWebMercator(nw).y - util.projectToWebMercator(sw).y) / util.WM_2R;
```

Comment:
I don't think this is a good idea either, the Renderer didn't know about webmercator or maps at all before

Reply:
Yes, I have decoupled the map so this functions are not in the Renderer anymore